runtime/src/bank.rs

```diff
@@ -5586,9 +5586,9 @@ impl Bank {
         &self,
         use_index: bool,
         mut debug_verify: bool,
-        slots_per_epoch: Option<Slot>,
         is_startup: bool,
     ) -> Hash {
+        let slots_per_epoch = Some(self.epoch_schedule().slots_per_epoch);
```

Comment:
@brooksprumo the value is retrieved from the bank here instead of being passed in. It is used in lower functions.

Reply:
Ah, duh, thanks!